packages/webview_flutter_lwe/pubspec.yaml

```
@@ -1,8 +1,8 @@
-name: webview_flutter_tizen
-description: Tizen implementation of the webview plugin
+name: webview_flutter_lwe
+description: Tizen implementation of the webview plugin using lwe
```

Longer description makes pub.dev happy.
```suggestion
description: Tizen implementation of the webview plugin backed by Lightweight Web Engine.
```